.github/workflows/main.yml

```diff
@@ -0,0 +1,40 @@
+name: Account Backup
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  schedule:
+    - cron: '*/5 * * * *'
```

Comment:
Use https://crontab.guru/examples.html to get the cron schedule expressions as required. Currently, I have set it to five minutes.